ARAnalytics.podspec

```diff
@@ -5,7 +5,7 @@ Pod::Spec.new do |s|
   s.homepage     =  'https://github.com/orta/ARAnalytics'
   s.authors      =  { 'orta' => 'orta.therox@gmail.com', 'Daniel Haight' => "confidence.designed@gmail.com" }
   s.source       =  { :git => 'https://github.com/orta/ARAnalytics.git', :tag => s.version.to_s }
-  s.ios.deployment_target = "7.0"
+  s.ios.deployment_target = "8.0"
```

Comment:
It turned into quite a chain of required changes. I also noted that `xctool` no longer does builds, so I needed to do this:

```
xcodebuild -workspace Example/ARAnalyticsTests.xcworkspace -scheme ARAnalyticsBootstrapiOS -sdk iphonesimulator
xctool -workspace Example/ARAnalyticsTests.xcworkspace -scheme ARAnalyticsBootstrapiOS -sdk iphonesimulator run-tests
```